_community_members/sb2k16.md

```diff
@@ -0,0 +1,24 @@
+---
+short_name: sb2k16
+name: Souvik Bose
+photo: '/assets/media/community/members/sb2k16.jpg'
+title: 'OpenSearch Community Member: Souvik Bose'
+primary_title: Souvik Bose
+breadcrumbs:
+  icon: community
+  items:
+    - title: Community
+      url: /community/index.html
+    - title: Members
+      url: /community/members/index.html
+    - title: 'Souvik Bose&apos;s Profile'
+      url: '/community/members/souvik-bose.html'
+github: sb2k16
+job_title_and_company: 'Software Development Engineer at AWS'
+personas:
+  - author
+permalink: '/community/members/souvik-bose.html'
+redirect_from: '/authors/sb2k16/'
+---
+
+Souvik Bose is a Software Development Engineer at AWS working in search services. He is a contributor to the Data Prepper project.
```

Comment:
```suggestion
Souvik Bose is a Software Development Engineer at AWS working on search services. He is a contributor to the Data Prepper project.
```

Reply:
Thanks @natebower. I have addressed this issue.